src/libigc/task.py

```
@@ -73,15 +80,18 @@ def __init__(self, turnpoints: list[Turnpoint], start_time: int, end_time: int):
-        """ Creates Task from LK8000 task file, which is in xml format.
-            LK8000 does not have End of Speed Section or task finish time.
-            For the goal, at the moment, Turnpoints can't handle goal cones or
-            lines, for this reason we default to goal_cylinder.
+        """Creates Task from LK8000 task file, which is in xml format.
+        LK8000 does not have End of Speed Section or task finish time.
+        For the goal, at the moment, Turnpoints can't handle goal cones or
+        lines, for this reason we default to goal_cylinder.
         """
 
         # Open XML document using minidom parser
         DOMTree = xml.dom.minidom.parse(filename)
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S318)